sound/soc/sof/sof-priv.h

```diff
@@ -221,6 +221,8 @@ struct snd_sof_control {
 	enum sof_ipc_ctrl_cmd cmd;
 	u32 *volume_table; /* volume table computed from tlv data*/
 
+	const char *vfe_link_name; /* name of Virtual FE link */
```

Comment:
how do we deal with > 1 VFEs ? Best to have a list.

Reply:
@lgirdwood but my understanding there would be multiple VFE's associated with different kcontrols. But one kcontrol would only ever be associated with only one VFE. Please correct me if I am wrong.